data/org.holylobster.nuntius.appdata.xml.in

```diff
@@ -1,22 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2015 Ignacio Casal Quinteiro <icq@gnome.org> -->
-<application>
-  <id type="desktop">org.holylobster.nuntius.desktop</id>
-  <metadata_license>CC0</metadata_license>
-  <project_license>GPL-2.0+</project_license>
+<component type="desktop-application">
+  <id>org.holylobster.nuntius.desktop</id>
+  <launchable type="desktop-id">org.holylobster.nuntius.desktop</launchable>
+  <name>Nuntius</name>
+  <summary>Nuntius delivers notifications from your phone or tablet to your computer</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
   <description>
-    <_p>
+    <p>
```

Comment:
Nuntius still uses intltool, so it needs the underscores.

Reply:
Thanks for the feedback. Fixed.

Reply:
Please also mark name, summary, and caption for translation.

Reply:
Done.

Reply:
Thanks a lot!